bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/cache/ExpiringCache.java

```diff
@@ -47,15 +49,12 @@ public ExpiringCache(long expiry, Supplier<V> action) {
 
```

Comment:
Imho you should remove this `null` check now

Reply:
Done.